### ISSUE TITLE
Make theme view interface

### ISSUE THREAD

**Issue by jtdaugherty** [matterhorn-chat/matterhorn#629]
This issue is about making an interface to visualize all available theme entries in an interactive way. Ultimately it would even be nice if that could let you change the entries in real time and persist them, but that is far enough from what's possible today that I'd like it to be a separate issue once the first part of this is done.

**Comment by jtdaugherty** [comment]
This is now essentially done on `develop` since that branch now has a patch to show a demonstration of every theme attribute on the `/help themes` page.
